tutorials/intro/Intro_Tutorial_4.ipynb

```diff
@@ -471,8 +478,8 @@
    },
    "outputs": [],
    "source": [
-    "from snorkel.learning import LogReg\n",
-    "disc_model = LogReg()"
+    "from snorkel.learning import LogisticRegression\n",
+    "disc_model = LogisticRegression()"
```

Comment:
Let's add it to another tutorial. LSTM will likely perform much worse on the intro tutorial than LR.